source-reminder/README.md

```diff
@@ -9,60 +9,55 @@ Thus, this tool exists to cross-reference articles on the website and articles s
 identify any pieces of work for which source information has not been submitted.
 
 ## Requirements
+* Docker
 * [Lambda](https://aws.amazon.com/free/?all-free-tier.sort-by=item.additionalFields.SortRank&all-free-tier.sort-order=asc&awsf.Free%20Tier%20Types=tier%23always-free) (Amazon Web Services)
 * [Google Drive](https://www.google.com/drive/)
-* [Slack SDK](https://slack.dev/node-slack-sdk/)
-* [Gspread](https://github.com/burnash/gspread) 
-* [Client library](https://github.com/googleapis/oauth2client) for OAuth 2.0&mdash;also known as `oauth2client`
-* [Python Requests](https://docs.python-requests.org/)
-* [Python Datetime](https://docs.python.org/3/library/datetime.html)
 
 _We can be loose with the requirements at times, but at least having them pinned to the major version of a package tends to be helpful._
-
-## Google Cloud Integration
-To properly receive the data from an API call, you&rsquo;ll be required at some point to provide what&rsquo;s called a *credentials* file for said account.
-It lives in the Amazon Web Services Lambda computing platform with the rest of the files and responds to webhook requests. Seven libraries are required, but two of them are pre-installed with Python, so really you only need to worry about packaging the five above.
-
+## Running Locally
+1. Clone repo and `cd source-reminder`
+2. Configure a local `docker-env` file with the needed environment variables
+    ```sh
+    cp docker-env-example docker-env
+    ```
+3. Run Source Reminder Lambda container
+    ```sh
+    # builds a local image "source-reminder/lambda:local"
+    # then runs a container from the image named "source-reminder" on localhost:9000
+    make run-container
+    ```
+4. From a new terminal window, invoke the lambda function (simulates a webhook request to execute Lambda)
+    ```sh
+    make test-container
+    # runs: curl -XPOST "http://localhost:9000/2015-03-31/functions/functioninvocations" -d '{}'
+    # In case of source-reminder, we send an empty data body in the request to invoke the function ("-d '{}'"), but for other lambda functions, like hoedown-helper, this data body can be configured to test different invokation scenarios.
+    ```
+5. The function's output will show up in the terminal running the `source-reminder` Lambda container (matches what would show up in AWS Cloudwatch logs on function invokation).
+6. _Optional_:  if `DRY_RUN=True`, no Slack messages will be sent.  The console will output what _would_ have been sent.  If you want to run the function locally and have it send the messages (running as production), then set `DRY_RUN=FALSE`.
```

Comment:
Before we do `make run-container` and `make test-container`, should we add `DRY_RUN=True` to the docker-env?

Reply:
Please ignore the comment😅 although maybe it could be a good idea to place `6` first, just so nobody runs make test-container before setting it as a DRY_RUN. What do you think? Or it could just be me that almost did.